### PR TITLE
Add bind and log uvicorn configuration

### PR DESCRIPTION
**Pull request by ioigoume**

### CODE REVIEW

**Review by lionick — APPROVED** [review]
LGTM

File: roles/rciam-metrics/defaults/main.yml [hunk 1]
```diff
@@ -11,6 +11,7 @@ metrics_api_url: http://localhost:8004/
 metrics_hostname: "metrics-dev.rciam.grnet.gr"
 metrics_protocol: "http"
 metrics_api_path: "/api/v1"
+fastapi_log_level: "info"
```
Comment (NicolasLiampotis) [inline roles/rciam-metrics/defaults/main.yml:14]: Do we expect log level configuration for other components of the metrics service? Is there a reason not go with a technology-neutral approach, e.g.:
```
metrics_api_log_level: "info"
metrics_api_syslog_identifier: "metrics_server_fastapi"
```

or to group log related variables:
```
metrics_api_logging:
  level: "info"
  syslog_identifier: "metrics_server_fastapi"
```

File: roles/rciam-metrics/templates/fastapi/fastapi.service.j2 [hunk 2]
```diff
@@ -8,7 +8,8 @@ Group=www-data
 
 WorkingDirectory={{ metrics_path }}
 Environment="PATH={{ metrics_path }}/.venv/bin"
-ExecStart={{ metrics_path }}/.venv/bin/gunicorn -w 4 -k uvicorn.workers.UvicornWorker app.main:app
+ExecStart={{ metrics_path }}/.venv/bin/gunicorn -w 4 -k uvicorn.workers.UvicornWorker app.main:app --log-syslog --log-syslog-prefix metrics_server_fastapi --bind {{ metrics_hostname }} --log-level {{ fastapi_log_level }}
```
Comment (NicolasLiampotis) [inline roles/rciam-metrics/templates/fastapi/fastapi.service.j2:11]: This depends on the previous comment. E.g. if we go with the `metrics_api_logging` approach:

`ExecStart={{ metrics_path }}/.venv/bin/gunicorn -w 4 -k uvicorn.workers.UvicornWorker app.main:app --log-syslog --log-syslog-prefix {{ metrics_api_logging.syslog_identifier }} --bind {{ metrics_hostname }} --log-level {{ {{ metrics_api_logging.level }} }}`

**Review by NicolasLiampotis — APPROVED** [review]
LGTM

**Pull request merged**